adhocracy4/comments/static/comments/react_comments.jsx

```diff
@@ -1,8 +1,11 @@
+import React from 'react'
+import { createRoot } from 'react-dom/client'
+
 const CommentBox = require('./CommentBox')
-const React = require('react')
-const ReactDOM = require('react-dom')
 
 module.exports.renderComment = function (el) {
   const props = JSON.parse(el.getAttribute('data-attributes'))
-  ReactDOM.render(<CommentBox {...props} />, el)
+  const container = el
```

Comment:
we could reduce one variable and either rename the parameter to `container` or using `el`?

Reply:
will do!